packages/fluentui/docs/src/examples/components/Card/Usage/CardExampleExpandable.tsx

```diff
@@ -0,0 +1,126 @@
+import {
+  Avatar,
+  Box,
+  BoxProps,
+  BoxStylesProps,
+  Card,
+  cardFocusableBehavior,
+  ComponentVariablesInput,
+  compose,
+  Flex,
+  Image,
+  Provider,
+  Text,
+  ThemeInput,
+} from '@fluentui/react-northstar';
+import * as React from 'react';
+
+// Compose custom styled Box
+type ExpandableBoxProps = {
+  expand?: boolean;
+};
+
+type ExpandableBoxStylesProps = ExpandableBoxProps;
+
+const ExpandableBox = compose<'button', ExpandableBoxProps, ExpandableBoxStylesProps, BoxProps, {}>(Box, {
+  className: 'ui-expandable-box',
+  displayName: 'ExpandableBox',
+  mapPropsToStylesProps: props => ({ expand: props.expand }),
+  handledProps: ['expand'],
+});
+
+//
+// Theme
+//
+
+type ComponentStylesProps = {
+  ExpandableBox: BoxStylesProps & ExpandableBoxStylesProps;
+};
+
+type ComponentVariables = {
+  ExpandableBox: {
+    restMaxHeight: string;
+    expandMaxHeight: string;
+  };
+};
+
+const componentStyles: {
+  // @ts-ignore TODO fix type there
+  [C in keyof ComponentStylesProps]: ComponentSlotStylesInput<ComponentStylesProps[C], ComponentVariables[C]>;
+} = {
+  ExpandableBox: {
+    root: ({ props: p, variables: v }) => ({
+      maxHeight: v.restMaxHeight,
+      transition: 'max-height 0.5s ease-in',
+      overflow: 'hidden',
+      ...(p.expand && {
+        maxHeight: v.expandMaxHeight,
+        transition: 'max-height 1s ease-in',
+      }),
+    }),
+  },
+};
+
+const componentVariables: ComponentVariablesInput = {
+  ExpandableBox: (): ComponentVariables['ExpandableBox'] => ({
+    restMaxHeight: '20px',
+    expandMaxHeight: '500px',
+  }),
+};
+
+const customTheme: ThemeInput<ComponentStylesProps> = {
+  componentStyles,
+  componentVariables,
+};
+
+const CardExampleExpandable = () => {
+  const [expandCard, setExpandCard] = React.useState(false);
+  const expandCardHandler = () => {
+    setExpandCard(true);
+  };
+
+  const shrinkCardHandler = () => {
```

Comment:
I don't think that there is any values in these callbacks. Let's remove them and inline, we can also use knobs, BTW